src/fonts/inter.scss

```diff
@@ -83,17 +83,16 @@ Usage:
 }
 
 /* Minimal font file with limited glyphs and layout features (only kern).
-   Should be enough for most en-US pages. This was generated by looking at
-   unicode ranges needed to display most of our en-US pages with glyphhanger
-   and manual tweaks to add several characters that are likely to appear.
+   Should be enough for most pages in en, de, fr, ru, es, pt, pl & it locales.
+   This was generated by looking at unicode ranges needed to display most of
+   pages with glyphhanger and manual tweaks to add several characters that are
+   likely to appear in those languages.
 
-   In serverHtml component, we preload subsets corresponding to the locale,
-   so a request for /en-US/something would preload this font because of its
-   name.
+   We preload any font that has `preload` in its name in serverHtml component.
 
    The corresponding command to generate the font is:
    pyftsubset Inter-roman.var.woff2
-    --output-file=Inter-roman-minimal-subset-en-US.var.woff2
+    --output-file=Inter-roman-subset-mainlocales.var.woff2
```

Comment:
I kinda liked the approach where each locale was added to the file name because "mainlocales" is quite obscure compared to `en_de_fr_ru_es_pt_pl_it`.

Reply:
Fair enough. This might cover more locales that the name implies, but at least which ones are the target would be explicit. I'll change it

Reply:
https://github.com/mozilla/addons-frontend/pull/10530/commits/2e911606816f5671b7724bb5c71ebdc55ec0575d